package.json

```diff
@@ -42,7 +42,7 @@
     "@radix-ui/react-visually-hidden": "^1.1.0",
     "@renegade-fi/internal-sdk": "0.4.12",
     "@renegade-fi/price-reporter": "0.0.0-canary-20250529193905",
-    "@renegade-fi/react": "0.0.0-canary-20250602165352",
```

Comment:
i believe i've approved the associated TS SDK PR so let's just cut a release and use the proper version